modules/rational-use-package.el

```diff
@@ -17,21 +17,22 @@
 ;; we are possibly here because `straight.el' is preferred, so only
 ;; configure package in that case. This work has already been done if
 ;; `package.el' is enabled at startup.
-(unless package-enable-at-startup
- (require 'package)
-
- ;; Emacs 27.x has the gnu elpa as the default for `package-archives'
- ;; Emacs 28.x has the gnu and nongnu elpa as the default.
- ;; Only add nongnu elpa for Emacs version < 28
- (when (version< emacs-version "28.0")
-   (add-to-list 'package-archives (cons "nongnu" "https://elpa.nongnu.org/nongnu/")))
-
- ;; Always add melpa to `package-archives'
- (add-to-list 'package-archives (cons "melpa" "https://melpa.org/packages/"))
-
- (package-initialize)
- (unless package-archive-contents
-   (package-refresh-contents)))
+(when (eq rational-packge-system 'package)
```

Comment:
We might want to update this part to use the `rational-package-install-package` command since `straight.el`users won't get `use-package` installed correctly without it.

Reply:
I think most of this can disappear since we are loading `rational-package` and it configures the stuff here in the case of package.el usage (thus redundant here) or in the case of straight, it isn't needed.